CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,128 @@
+# Contributor Covenant Code of Conduct
```

Comment:
I think this needs to be discussed at the MESSAGE team meeting, and everyone in the team must (a) agree to abide by this and (b) mention if there are other/additional points that they think should be included in order for them to feel the community is genuinely "open, welcoming, etc."

Perhaps we can hold off on merging this PR until we can have such a discussion. To inform it, people not familiar with CoCs might need some information like:
- What kind of things is a CoC intended to help prevent, and how?
- Are there different kinds of CoCs?
- Why do we choose this one in particular?